content/ja/handbook/engineering/ai/agent-foundations/index.md

```
@@ -0,0 +1,179 @@
+
+私たちは Issue が作業されるべき順序の理解を助けるためにラベルを使用します。
+
+1. **Deliverable**: これらのアイテムはイテレーションの主要な成果物であり、最初に取り上げられるべきです。
+1. **Stretch**: これらのアイテムのほとんどを提供することを目指していますが、野心的な計画の一環として、それらの一部は遅れる可能性があります。
+
+## 👏 コミュニケーション
+
+Agent Foundations チームは、以下のガイドラインに基づいてコミュニケーションを行います。
+
+1. 同期ミーティングよりも常に非同期コミュニケーションを優先する。
+1. 非同期が非効率になっている場合、[同期コール](#-ad-hoc-sync-calls) の手配を躊躇しないでください。ただし、チームメンバーと共有するために常に録画してください。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)